### DISCUSSION TITLE
希望可以针对早期视频为弹幕转义添加对/n的转义支持

### DISCUSSION THREAD

**Discussion by atoposyz** [the1812/Bilibili-Evolved#4865] [功能建议 / Ideas]
部分早期视频的高级弹幕使用/n进行换行，但是现在已经失效，例如[BV1Yx411c71s](https://www.bilibili.com/video/BV1Yx411c71s/)。希望可以增加对`/n`的转义支持。

![QQ20240809-121641](https://github.com/user-attachments/assets/fd55bda3-094c-4f01-a218-dbb5edab858c)

**Comment by Geeyun-JY3** [comment]
关联：#3470
